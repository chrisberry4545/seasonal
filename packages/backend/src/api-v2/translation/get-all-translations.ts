import { translationsEn } from './translations-en';

export const getAllTranslations = (): any => ({
  en: translationsEn
});
