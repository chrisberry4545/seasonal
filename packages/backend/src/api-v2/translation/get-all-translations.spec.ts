import { getAllTranslations } from './get-all-translations';

describe('getAllTranslations', () => {
  test('matches snapshot', () => expect(getAllTranslations()).toMatchSnapshot());
});
