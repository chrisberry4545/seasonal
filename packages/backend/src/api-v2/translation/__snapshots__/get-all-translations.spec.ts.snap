// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getAllTranslations matches snapshot 1`] = `
Object {
  "en": Object {
    "dietFiltersAll": "All",
    "dietFiltersVegan": "Vegan",
    "dietFiltersVegetarian": "Vegetarian",
    "en-AU": "English (Australian)",
    "en-CA": "English (Canadian)",
    "en-GB": "English (British)",
    "en-NZ": "English (New Zealand)",
    "en-US": "English (USA)",
    "feedbackDoYouLikeTheAppQuestion": "Is Eat Seasonal useful?",
    "feedbackImprovementQuestion": "We'd love to hear feedback on how we could improve",
    "feedbackImprovementsPlaceholder": "Your feedback",
    "feedbackRateOnStoreQuestion": "Rating the app will help other people find the app and start eating seasonally",
    "feedbackRateOnStoreQuestionTitle": "Help others Eat Seasonal",
    "feedbackRateTheApp": "Rate the app",
    "feedbackSend": "Send feedback",
    "foodBadgesTitle": "Nutrients",
    "foodForBadgeFoundIn": "Found in",
    "foodTab": "Food",
    "fr-FR": "French",
    "gridView": "Grid view",
    "listView": "List view",
    "menuAboutUs": "About us",
    "menuAllSeasons": "All seasons",
    "menuChangeRegion": "Change region or language",
    "menuGiveFeedback": "Give feedback",
    "no": "No",
    "notNow": "Not now",
    "privacyPolicyTitle": "Privacy Policy",
    "recipesForFoodTitle": "Recipes",
    "recipesTab": "Recipes",
    "regionChangedDetectedYourClosestRegionAs": "We've detected your closest region as:",
    "regionChangedIfThisIsWrongClickHere": "If this is wrong, click here.",
    "searchBarPlaceholder": "Search",
    "seasonsForFoodKeyInSeason": "In season",
    "seasonsForFoodKeyNotInSeason": "Not in season",
    "seasonsForFoodTitle": "When are they in season?",
    "selectYourLanguage": "Select your language",
    "selectYourRegion": "Select your region",
    "yes": "Yes",
  },
}
`;
